backstory/src/demo/webapp/gpt2.js

```diff
@@ -17,3 +16,74 @@
  */
 
 // DISPLAY STORY
+
+async function displayStory() {
+  /* eslint-disable no-unused-vars */
```

Comment:
Can you use this just to wrap the single unused function declaration? Like this:
https://github.com/eslint/eslint/issues/1939#issuecomment-77225110

Reply:
Done